thunx-autoconfigure/build.gradle

```diff
@@ -15,7 +15,6 @@ dependencies {
 
     compileOnly project(':thunx-spring')
     compileOnly project(':thunx-pdp-opa')
-    compileOnly "eu.xenit.contentcloud:opa-async-java-client:0.3.0"
```

Comment:
No longer needed in the autoconfigure?

Reply:
it's transitively imported with  `compileOnly project(':thunx-pdp-opa')`

Reply:
To answer your question directly: no, it was not needed in the first place ;-)